### ISSUE TITLE
Issue : error when passing n_estimators parameter to rf

### ISSUE THREAD

**Issue by danbricedatascience** [pycaret/pycaret#606]
Error when passing `n_estimators` to `rf` model.

```python
clf = create_model('rf',fold=5,
                   class_weight={0:1,1:5},
                   n_estimators=5,
                   max_depth=5)
```

We get this error: 

```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-90-41e6deb69c6b> in <module>
----> 1 clf = create_model('rf',fold=5,
      2                    class_weight={0:1,1:5},
      3                    n_estimators=5,
      4                    max_depth=5)

~/anaconda3/envs/ai/lib/python3.8/site-packages/pycaret/classification.py in create_model(estimator, ensemble, method, fold, round, cross_validation, verbose, system, **kwargs)
   3118 
   3119         from sklearn.ensemble import RandomForestClassifier
-> 3120         model = RandomForestClassifier(n_estimators=10, random_state=seed, n_jobs=n_jobs_param, **kwargs)
   3121         full_name = 'Random Forest Classifier'
   3122 

TypeError: ABCMeta object got multiple values for keyword argument 'n_estimators'
```

**Comment by Yard1** [comment]
Will be fixed in 2.2.

**Comment by pycaret** [comment]
@danbricedatascience Thanks. Looks like a bug. We will fix this in the next release of `pycaret` (expect at the end of Sep).

Thanks.

**Comment by pycaret** [comment]
@danbricedatascience 

This issue has been fixed in the `pycaret-nightly` release. Please install pycaret-nightly using `pip install pycaret-nightly`. Feel free to open a new issue, in case the problem persists. This fix will be published to PyPi as `pycaret==2.2.0` on October 28, 2020.
